apiary.apib

```diff
@@ -291,6 +291,8 @@ Get a list of departments.
             + (object)
                 + id: `67c576e7-7e6f-465d-b6ab-a864f6e5e95b` (string)
                 + name: `Human Resources` (string)
+                + vat_number: `BE0899623035` (string)
+                + website: `https://piedpiper.com` (string)
```

Comment:
Q: why is this necessary in the `list`? My concern is that this field is not required and I would imagine that most of the times it will be `null`?

Reply:
I can imagine that you'd like to show that in an overview of departments?

Reply:
Okay, you're right, website is filled in for only 15% of departments and is not really prominent in the UI, so it'll be `null` a lot of the times.

Not sure if this is an issue though? I can still imagine this getting a more prominent place later?

Reply:
We could still add the fields at that moment though.

Reply:
The thing is if it's only 15% than the UI will have an empty column 85% of the times will be awkward for the user IMO other than that having the field would be ok ;)